requirements.txt

```diff
@@ -1,3 +1,3 @@
-torch>=2.1.0
-lightning @ git+https://github.com/Lightning-AI/lightning@2b6b594dabe668752183f5ab8a1d93d0904c4d32
+torch==2.2.0.dev20231026
```

Comment:
Why does min torch need to increase? There was a lot of complains about having to install nightly in the past. If this is optional (for instance for compile support) then I would use `_TORCH_GREATER_EQUAL` etc in the code

Reply:
Ok chill 😅 No big deal. 
We need it for torch.compile, remember how we debugged that sheet together? hehe 
So I'll include an instruction in the tutorial to install nightly. I'd like to avoid the `_TORCH_GREATER_EQUAL` boilerplate in user scripts.